test/extended/router/scoped.go

```diff
@@ -17,6 +17,7 @@ import (
 	e2e "k8s.io/kubernetes/test/e2e/framework"
 
 	routeclientset "github.com/openshift/client-go/route/clientset/versioned"
+	"github.com/openshift/origin/pkg/oc/lib/routedisplayhelpers"
 	routeapi "github.com/openshift/origin/pkg/route/apis/route"
```

Comment:
Use routev1 instead of routeapi?
```
- routeapi "github.com/openshift/origin/pkg/route/apis/route"
+ routev1 "github.com/openshift/api/route/v1"
```
 and then replace routeapi with routev1 in the file

Reply:
> Use routev1 instead of routeapi?
> 
> - routeapi "github.com/openshift/origin/pkg/route/apis/route"
> + routev1 "github.com/openshift/api/route/v1"
> and then replace routeapi with routev1 in the file

We'll get that during a future sweep.  Often we see ripples coming out from changes like that, so we try to find the smallest useful chunk we can bound (I know it doesn't look that way with 3k lines, but it was the best I could do)